src/hooks/engine.rs

```diff
@@ -321,6 +321,34 @@ pub static paintbuffer: Pointer<*mut [portable_samplepair_t; 1026]> =
     Pointer::empty(b"paintbuffer\0");
 pub static paintedtime: Pointer<*mut c_int> = Pointer::empty(b"paintedtime\0");
 pub static realtime: Pointer<*mut f64> = Pointer::empty(b"realtime\0");
+
+pub static R_DrawSequentialPoly: Pointer<
+    unsafe extern "C" fn(*mut c_void, *mut c_int) -> *mut c_void,
+> = Pointer::empty_patterns(
+    b"R_DrawSequentialPoly\0",
+    // Stolen from BunnymodXT
```

Comment:
I know this is effort but I want to keep "how to find" descriptions on all patterns. Would it be possible for you to write them, by figuring out how to find the functions in Ghidra?